sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -94,4 +172,14 @@ public partial class RoleDefinition
         public string RoleType { get { throw null; } }
         public string Type { get { throw null; } }
     }
+    public partial class SelectiveKeyRestoreDetails
```

Comment:
I've already looped back with the service team to generalize the fullrestore* models and operations to restore, since other than the start of the operation the status methods are applicable to selective restore also. 

Is there a reason why we'd not want to expose selectiveRestore?

Reply:
never mind - saw your other comment :)